### PR TITLE
More plugins

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Comment by Fobhep** [comment]
This needs a rebase, since I merged a newer "no-brainer" - PR first 
sorry for the inconvenience

File: sop/plugins/clip.py [hunk 1]
```diff
@@ -10,9 +10,22 @@ def sop_plugin():
...
+            self.message = message
+
+        def handle(self):
+            pyperclip.copy(self.message)
+
+        def msg(self):
+            return "Copy barcode to clipboard."
+
     def __init__(self):
         logging.debug("Clip aktiviert")
```
Comment (Fobhep) [inline sop/plugins/clip.py]: Make this english please

File: sop/plugins/stdout.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            self.message = message
+
+        def handle(self):
+            print(self.message)
+
+        def msg(self):
+            return "Print barcode to stdout."
+
+    def __init__(self):
+        logging.debug("Out aktiviert")
```
Comment (Fobhep) [inline sop/plugins/stdout.py]: In english please :)

File: sop/plugins/url.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+        def handle(self):
+            subprocess.run(['sensible-browser', self.message])
+
+        def msg(self):
+            return "Open barcode in web-browser. ('{}')".format(self.message)
+
+    def __init__(self):
+        self.matcher = re.compile(self.match_pattern)
+        logging.debug("QrUrl aktiviert")
```
Comment (Fobhep) [inline sop/plugins/url.py]: in english please

**Comment by mdellweg** [comment]
@Fobhep rbsd.

**Pull request merged**